h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java

```diff
@@ -144,6 +144,7 @@ public enum Backend {
     public Backend _backend = Backend.auto;
 
     public String _eval_metric;
+    boolean _score_eval_metric_only;
```

Comment:
Some leftover?

Reply:
This parameter is needed but it should be public - thanks for the comment.